recipe/meta.yaml

```diff
@@ -18,7 +18,7 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.5
+    - python
     - pytorch
```

Comment:
Is `pytorch`  necessary host dependency? Should we not use `--no-deps` so that pip and python is sufficient? If that is not the case, then we should make sure the `pytorch` is identical between `host` and `run`